planning/behavior_path_planner/include/behavior_path_planner/scene_module/avoidance/avoidance_module.hpp

```diff
@@ -62,9 +62,9 @@ class AvoidanceModule : public SceneModuleInterface
   void updateData() override;
   void acceptVisitor(const std::shared_ptr<SceneModuleVisitor> & visitor) const override;
 
-  void updateModuleParams(const std::shared_ptr<AvoidanceParameters> & parameters)
+  void updateModuleParams(const std::any & parameters) override
   {
-    parameters_ = parameters;
+    parameters_ = std::any_cast<std::shared_ptr<AvoidanceParameters>>(parameters);
```

Comment:
How about moving this function to SceneModuleInterface by adding a template of `ModuleParameters` which will be AvoidanceParameters, DynamicAvoidanceParameters, etc.
Or any reason to define the function in the specific scene module side?

Reply:
In my understanding, we can't define `parameter_` in interface side because it depends on each modules. So, I define `parameter_` and this function in inherit class side.

Or, you mean you think it is better to add base parameter struct and create inherit parameter struct (e.g. AvoidanceParameter) from it?